C/guide/appendixd.xml

```diff
@@ -758,7 +758,7 @@ Blocking_Chars</literallayout>
                     <guimenu>File</guimenu><guimenuitem>Properties </guimenuitem>
                   </menuchoice>
                   for number source (transaction number or anchor-split action - see
-                  <ulink url="&url-docs-C;help/book-options.html#num-action-book-option">Use Split
+                  <ulink url="&url-docs-C;manual/book-options.html#num-action-book-option">Use Split
```

Comment:
I wonder, if we should delay the rename in &url-docs*; until @derekatkins is back and fixed the docs build process on code.gnucash.org.
[Edit:]
The current state on code is from 2023-03-24.
[Todays IRC](https://lists.gnucash.org/logs/2023/07/23.html#T16:25:45)

Reply:
I just realized, code is still on V4. So I will change docs to docs-released. That would be more recent.